modules/standard/Sys.chpl

```diff
@@ -58,6 +58,11 @@ module Sys {
   // C code generation.
   // mferguson -- TODO -- is that still true? Even for LLVM?
 
+  // fcntl command flags
+  extern const F_GETFL:c_int;
```

Comment:
Some of these constants, such as `S_IRWXU`, do not work with the LLVM backend. So it's important to test that these constants do work with the LLVM backend. If they don't, we will create C wrapper functions and parenless Chapel functions to call them (e.g. `proc S_IRWXU { return chpl_wrap_S_IRWXU(); }` )

Reply:
My current build for chapel uses LLVM Backend it works there, or shall I check in some other way

Reply:
That's fine